app/bundles/CoreBundle/Helper/Chart/SeriesPieChart.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace Mautic\CoreBundle\Helper\Chart;
+
+class SeriesPieChart extends AbstractChart implements ChartInterface
+{
+    /**
+     * Holds the suma of the all dataset values.
+     *
+     * @var float
+     */
+    protected $totalCount = 0;
```

Comment:
```suggestion
     */
    protected float $totalCount = 0;
```

Reply:
I changed to

`protected int|float $totalCount = 0;`